domains/qloak.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "qloakongithub",
+        "email": "7469931481@stu.gcsnc.com"
+    },
+    "record": {
+        "CNAME": "qloakongithub.github.io/qloak-website/"
```

Comment:
Site gives an error, also record is invalid
```suggestion
        "CNAME": "qloakongithub.github.io"
```

Reply:
Ignore this.

Reply:
If you don't need this one then you can delete the json file

Reply:
Alright.